server/test/stubs/README.md

```diff
@@ -0,0 +1,23 @@
+# Stubbing Entities
+
+## Return a new instance every time
+
+The stubs need to return a new object every time to avoid referencing the same instance:
+
+Ok:
+
+```typescript
+export const sampleStub = (): SampleEntity => {
+  return {
+    prop1: 'fakeval',
+  };
+};
+```
+
+Will run into problems:
+
+```typescript
+export const sampleStub: SampleEntity = {
+  prop1: 'fakeval',
+};
+```
```

Comment:
what problems might we run into?
if you're worried about this object getting changed, we actually shouldn't be changing it. we should take care to have pure functions. so maybe using objects would be better that way it would expose any side effect manipulation happening in our tested functions?
just a thought, we can discuss and make changes later